app/policies/comment_policy.rb

```diff
@@ -0,0 +1,36 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :user, :comment
+
+  def initialize(user, comment)
+    @user = user
+    @comment = comment
+  end
+
+  def index?
```

Comment:
nitpick: we are inheriting from application policy so no need to add false. that is already the default